py3status/modules/sysdata.py

```diff
@@ -190,44 +207,68 @@ def deprecate_function(config):
                         'param': 'med_threshold',
                         'msg': 'obsolete, set using thresholds parameter',
                         },
-                    ]
+                    {
+                        'param': 'padding',
+                        'msg': 'obsolete, use the format_* parameters',
+                        },
+                    {
+                        'param': 'precision',
+                        'msg': 'obsolete, use the format_* parameters',
+                        },
+                    ],
+                'update_placeholder_format': [
+                    {
+                        'function': update_deprecated_placeholder_format,
+                        'format_strings': ['format']
+                        },
+                    ],
+                }
+
+        update_config = {
+                'update_placeholder_format': [
+                    {
+                        'placeholder_formats': {
+                            'cpu_usage': ':.2f',
+                            'cpu_temp': ':.2f',
+                            'mem_total': ':.2f',
+                            'mem_used': ':.2f',
+                            'mem_used_percent': ':.2f',
+                            },
+                        'format_strings': ['format']
+                        },
+                    ],
                 }
 
     def post_config_hook(self):
         self.data = GetData(self)
         self.cpu_total = 0
         self.cpu_idle = 0
-        self.values = {}
+        self.values = {'temp_unit': self.temp_unit}
```

Comment:
This is incorrect imho.

If I have a `temp_unit = 'c'`  then `{temp_unit}` should be `°C` etc